amundsen_application/static/js/interfaces/Analytics.ts

```diff
@@ -1,4 +1,4 @@
 export interface AnalyticsEvent {
   name: string;
-  properties: { [prop: string]: unknown };
+  payload: { [prop: string]: unknown };
```

Comment:
curious why this change? `payload` already has meaning elsewhere, `properties` isn't great but it is the normal thing in analyitcs

Reply:
Yeah, I changed it because it wasn't getting mapped on the analytics middleware, as we were expecting payload.

Here we are using the redux style of name/payload, so the meta.analytics is a redux action as well.
We kept the properties denomination on the last step (trackEvent)

Reply:
ahhh gotcha